pandas/core/groupby/generic.py

```diff
@@ -1056,9 +1058,47 @@ def _cython_agg_blocks(
             result = no_result
             locs = block.mgr_locs.as_array
             try:
-                result, _ = self.grouper.aggregate(
-                    block.values, how, axis=1, min_count=min_count
+
```

Comment:
please don't do this here. this patch needs to be much lower in the stack. follow .aggregate thru and see where other types of groupers are handled.

Reply:
this is two calls further in the stack (where the NotImplementedError is raised to start with). look like a better place for this patch?
https://github.com/pandas-dev/pandas/blob/1c0cc62e3/pandas/core/groupby/ops.py#L455

Reply:
Moved in commit `0340554e2`